question/templates/qbank_columnsortorder.mustache

```diff
@@ -0,0 +1,47 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template core_admin/setting_qbanksetting.mustache
+
+    Admin question bank setting template.
+
+    Context variables required for this template:
+    * name - plugin name
+
+    Example context (json):
+    {
+        "name": "createcourse",
+    }
+}}
+
+<h2>
+    {{header}}
+</h2>
+<ul class="list list-group" id="sortable">
+    {{#names}}
+        <li class="item list-group-item list-group-item-action">
+            {{>core/drag_handle}}
+            <span class="retrievable">
+                {{.}}
```

Comment:
I know this is working because is just one parameter, but It's better to use the specific parameter name in case we need to add more information in the array.

Reply:
Can you changes this as requested above please?, I know it's something little, but it's easier to follow in that way.